extractors/paths/url_extraction/config.py

```diff
@@ -15,8 +15,12 @@ def get_config():
         available_for=["refinery", "common"],
         part_of_group=[
             "paths",
+            "personal_identifiers"
```

Comment:
is the order of the groups correct? this would now be sorted in paths not pii if i can trust the comment

Reply:
Yes, the first group is always the parent directory. When the URL extraction brick was built, it was put into the "paths" group. However, the cognition Notion pages states that the URL should be used for PII as well, so I added the group here.

Reply:
works for me, though maybe this should have the main group personal_identifiers

Reply:
I suggest creating an issue and tackling the group change at a later point intime

Reply:
Created an issue for this here: https://github.com/code-kern-ai/bricks/issues/386
Resolving conversation now